main.tf

```diff
@@ -26,9 +26,67 @@ resource "aws_grafana_workspace" "this" {
   role_arn                  = var.create_iam_role ? aws_iam_role.this[0].arn : var.iam_role_arn
   stack_set_name            = coalesce(var.stack_set_name, var.name)
 
+  dynamic "vpc_configuration" {
+    for_each = length(var.vpc_configuration) > 0 ? [var.vpc_configuration] : []
+
+    content {
+      security_group_ids = var.create_security_group ? concat([aws_security_group.this[0].id], vpc_configuration.value.security_group_ids) : vpc_configuration.value.security_group_ids
```

Comment:
Is there a reason why `vpc_configuration.value.security_group_ids` is a mandatory field when creating security groups? 

Now it's needed to set an empty list while I would prefer to leave out `security_group_ids` as I intend to to create security groups. So: 

```
  create_security_group = true

  vpc_configuration = {
    subnet_ids = var.subnet_ids
    security_group_ids = []
  }
```

Reply:
but thank you for the reminder to finish testing this 😅

Reply:
hehe thanks for the PR!  🥇